hack/tools/release/notes/print.go

```diff
@@ -167,6 +171,19 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 		}
 	}
 
+	// Print the list of dependencies
+	fmt.Println("")
+
+	repoURL := fmt.Sprintf("https://github.com/%s", p.repo)
+	deps, err := notes.NewDependencies().ChangesForURL(
+		repoURL, p.fromTag, p.toTag,
+	)
+	if err != nil {
+		fmt.Printf("Error getting dependencies: %v", err)
```

Comment:
I would add a new error return parameter and propagate it up (with a bit of error wrapping).

Then we can fail properly